pybeautifyjson.fish

```diff
@@ -1,5 +1,7 @@
-# beautify json string
-# use : pybeautifyjson '{"foo": "lorem", "bar": "ipsum"}'
-function pybeautifyjson
-  echo $argv | _python -mjson.tool
+function pybeautifyjson -d "Beautify a JSON string"
```

Comment:
Unrelated to this PR, in the future we could change this function to something smaller like `beautyjson`, which is 10 chars vs 14.